src/app/clusters/ota-server/ota-server.cpp

```diff
@@ -56,7 +98,24 @@ bool emberAfOtaSoftwareUpdateServerClusterApplyUpdateRequestCallback(app::Comman
 bool emberAfOtaSoftwareUpdateServerClusterNotifyUpdateAppliedCallback(app::Command * commandObj, chip::ByteSpan updateToken,
                                                                       uint32_t currentVersion)
 {
-    return false;
+    EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
+
+    ChipLogDetail(Zcl, "OTA Server received NotifyUpdateUpplied");
+
+    if (updateToken.size() != kUpdateTokenParamLength)
+    {
+        emberAfSendImmediateDefaultResponse(EMBER_ZCL_STATUS_INVALID_FIELD);
```

Comment:
This should probably be INVALID_ARGUMENT

Reply:
There is no EMBER_ZCL_STATUS_INVALID_ARGUMENT. Is there something else you'd like me to use? https://github.com/project-chip/connectedhomeip/blob/4f0ade5fcd50f99f53188ec401e18c2a22f47e14/src/app/util/af-enums.h#L23

Reply:
You have to add it, since Ember library predates Matter and doesn't have it yet.

Reply:
I can add it.

Just to clarify - are you suggested that EMBER_STATUS_ errors also reflect all CHIP_ERROR_ types?

Reply:
Also, what should EMBER_ZCL_STATUS_INVALID_FIELD be used for then?